src/frontend/Sider/Behandling/Vilkårvurdering/tekster.ts

```diff
@@ -11,6 +11,15 @@ export const svarIdTilTekst: Record<string, string> = {
         'Ja, tiltak/utdanningssted har dokumentert at søker er borte fra hjemmet utover vanlig arbeidstid',
 };
 
+export const svarIdTilTekstKortversjon: Record<string, string> = {
+    JA: 'Ja',
+    NEI: 'Nei',
+
+    // PASS_BARN
+    TRENGER_MER_TILSYN_ENN_JEVNALDRENDE: 'Ja',
+    FORSØRGER_HAR_LANGVARIG_ELLER_UREGELMESSIG_ARBEIDSTID: 'Ja',
```

Comment:
Å kun vise "Ja" på unntaket føler jeg viser for lite informasjon. Her burde man nog vise hva unntaket er?